data/Makefile.am

```
@@ -8,7 +8,13 @@ desktop_DATA = \
 	$(NULL)
 
 icon64dir = $(datadir)/icons/hicolor/64x64/apps
-icon64_DATA = eos-google-chrome.png
+icon64_DATA = icons/hicolor/64x64/eos-google-chrome.png
```

```suggestion
dist_icon64_DATA = icons/hicolor/64x64/eos-google-chrome.png
```

And drop the reference to `$(icon64_DATA)` from `EXTRA_DIST` below.

I think.

It’s been a while, autotools.

---

> I built the package but did not distcheck.

That’s why we have Jenkins do that for us, no?